app/views/scaptimony_policies/disassociate_multiple_hosts.html.erb

```diff
@@ -0,0 +1,7 @@
+<%= render 'hosts/selected_hosts', :hosts => @hosts %>
+<%= form_for :policy,
+             :url => remove_policy_from_multiple_hosts_scaptimony_policies_path(:host_ids => params[:host_ids]) do |f| %>
+    <%= selectable_f f, :id, [[_("Select Compliance Policy"), "disabled"],
```

Comment:
Just out of curiosity, I suppose we can't use :placeholder here because of the disabled values? Also if possible try to use `''` instead of `""` when there's no interpolation in strings.